src/content/glossary/index.md

```diff
@@ -424,6 +430,18 @@ A procedural (imperative) programming language with syntax that is similar to Ja
 
 A [hard fork](#hard-fork) of the Ethereum blockchain, which occurred at block 2,675,000 to address more denial-of-service attack vectors and clear state (see [Tangerine Whistle](#tangerine-whistle)). Also, a replay attack protection mechanism.
 
+### stablecoin {#stablecoin}
+
+An [ERC-20 token](#token-standard), backed by a reserve asset, with price pegged to real-world asset value. Examples of uses include pegging an ERC-20 tokens value to the value of fiat currency, precious metals, or other cryptocurrencies.
```

Comment:
Suggested edit:

"An ERC-20 token with a value pegged to another asset's value. There are stablecoins backed by fiat currency like dollars, precious metals like gold, and other cryptocurrencies like Bitcoin."